public_html/wp-content/plugins/pattern-directory/includes/admin-patterns.php

```diff
@@ -90,6 +92,17 @@ function pattern_list_table_render_custom_columns( $column_name, $post_id ) {
 				echo '&mdash;';
 			}
 			break;
+
+		case 'language':
+			$locale        = $current_pattern->wpop_locale ?: 'en_US';
+			$locale_labels = get_locales_with_english_names();
+
+			if ( isset( $locale_labels[ $locale ] ) ) {
+				echo esc_html( $locale_labels[ $locale ] );
+			} else {
+				echo '&mdash;';
```

Comment:
Is this case something we should be concerned about? A locale property exists but isn't found in the registered list?

Reply:
It's unlikely, but it could theoretically happen if our locale data from GlotPress changes.